### ISSUE TITLE
Data Library - Expand metadata

### ISSUE THREAD

**Issue by fvankrieken** [NYCPlanning/data-engineering#498]
Bit of a stub for now - this issue is contained within other issues as well since our templates will define ingestion, any transformations needed, potentially data validation, etc, but a few main points:
- data expectations. Questions along the line of (for a specific source dataset)
  - What is expected schema? 
  - What columns do we need? (warn vs error when dropped?) 
  - What preprocessing/renaming/cleaning/filtering steps occur? This obviously is a bit blocked by reorganizing code a bit
  - How frequently do we expect to update it?
- we should maybe follow some sort of metadata standard. I know Tyler has looked into this a bit, but we can certainly strive for some consistency between how we're storing metadata for source datasets as well as datasets we produce, to the point that it makes sense. And then, do we use that form of metadata explicitly as input to extract code? Or is some subset generated from the single source of truth metadata?

Going to leave cleanup of current "source" logic as a separate issue for now, though these could merge if we decide to start writing this code from scratch so-to-speak instead of modifying current data library

**Comment by fvankrieken** [comment]
Putting this as blocked for now, really it will happen as part of #499, #631, #651, #652. I still think it's worth having this issue in case we end up using it as a spot to discuss the metadata/template format

**Comment by fvankrieken** [comment]
Going to close - it really is something that's better handled as part of other linked issues - what's needed to ingest a dataset, what's needed to parse its geographic data, what are our expectations of the data. Not hugely useful as a standalone issue
